docker/postgres/docker-entrypoint-initdb.d/mock-data/interfaith_DDL.sql

```diff
@@ -54,21 +54,21 @@ CREATE TABLE needs (
   need_active BOOLEAN NOT NULL
 );
 
-DROP TABLE IF EXISTS needcallouts cascade;
-CREATE TABLE needcallouts (
-  needcallout_id BIGSERIAL PRIMARY KEY,
-  needcallout_quantity INT NOT NULL,
+DROP TABLE IF EXISTS calloutneeds cascade;
```

Comment:
I like the renaming here since a callout can have multiple needs, it seems more intuitive